api/runes/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@hirosystems/runes-api",
```

The ordinals api has the name updated to this format
```suggestion
  "name": "runes-api",
```